src/test/scala/uk/gov/hmrc/perftests/dds/Configuration.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.perftests.dds
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import io.gatling.http.check.HttpCheck
+import uk.gov.hmrc.performance.conf.ServicesConfiguration
+
+trait Configuration extends ServicesConfiguration {
+
+  val authLoginStubUrl: String     = baseUrlFor("auth-login-stub")
+  val addressLookupUrl: String     = baseUrlFor("address-lookup")
+  val digitalDisclosureUrl: String = baseUrlFor("digital-disclosure")
+
+  private val csrfTokenPattern: String              = """<input type="hidden" name="csrfToken"\s+value="([^"]+)""""
```

Comment:
Re-usable check patterns and functions should be abstracted to the configuration like this. If there is ever a change to any page affecting your re-used checks you only have to update one line, rather than every single check on every single request.